### ISSUE TITLE
Typo In Table?

### ISSUE THREAD

**Issue by ldkong1205** [dvlab-research/SphereFormer#1]
Hi @X-Lai, thanks for open-sourcing your work!

I noticed from the repository that your model has a mIoU of 67.8 on the SemanticKITTI validation set; I also read from your paper that SphereFormer is achieving 74.8 mIoU on the test set. I wonder if there is a misalign between these two numbers because usually the val results and test results should not have a gap that is that huge ... Do I miss anything from the paper? Can you tell me how you got the score like this? Thanks.

**Comment by X-Lai** [comment]
Thanks for your interest in our work. 

It is notable that 67.8% mIoU is the single-forward performance. For testing, we follow many previous works to use various tricks to boost the performance, such as testing-time augmentation, gt instance sampling, and multi-model ensembling.

**Comment by ldkong1205** [comment]
Hi @X-Lai, can you elaborate more on these aspects? As we would like to reproduce the scores of your models. Thanks.

**Comment by X-Lai** [comment]
1) for testing-time augmentation, the code has already been released, so you can directly follow it;
2) for gt instance sampling, we select several tail classes and record the `xyz` of all their instances before training. During training, we randomly sample several instances and paste them into the current scene. For more detail, we recommend to refer to Sec. 3.4 of the paper of RPVNet.
3) For multi-model ensembling, we train the model with different hyper-parameters such as weight decay, learning rate, etc. And finally average the predictions of them during inference.

Hope this can help you.

**Comment by ldkong1205** [comment]
Hi @X-Lai, your reply is helpful. I will try these details when reproducing your results.

**Comment by WWW2323** [comment]
Hi, @X-Lai , does SphereFormer also use these tricks for test results on NuScenes? or are there some differences between SemanticKITTI and NuScenes?

**Comment by sshan-zhao** [comment]
> 1. for testing-time augmentation, the code has already been released, so you can directly follow it;
> 2. for gt instance sampling, we select several tail classes and record the `xyz` of all their instances before training. During training, we randomly sample several instances and paste them into the current scene. For more detail, we recommend to refer to Sec. 3.4 of the paper of RPVNet.
> 3. For multi-model ensembling, we train the model with different hyper-parameters such as weight decay, learning rate, etc. And finally average the predictions of them during inference.
> 
> Hope this can help you.

Hi, @X-Lai for submission, do you train the model on the combination of training and validation set or only the training set? Thank you.

**Comment by X-Lai** [comment]
@sshan-zhao Yes, we follow previous works to use both training and validation sets for training in the submission.

**Comment by L-Reichardt** [comment]
For reference:
Panoptic-PolarNet has an GT-Instance Augmentation implementation [here](https://github.com/edwardzhou130/Panoptic-PolarNet/blob/main/dataloader/instance_augmentation.py). It seems to function like what is described in RPVNet and extracts the instances into a pickle file prior to training.

**Comment by Xavier-wa** [comment]
> 1. for testing-time augmentation, the code has already been released, so you can directly follow it;
> 2. for gt instance sampling, we select several tail classes and record the `xyz` of all their instances before training. During training, we randomly sample several instances and paste them into the current scene. For more detail, we recommend to refer to Sec. 3.4 of the paper of RPVNet.
> 3. For multi-model ensembling, we train the model with different hyper-parameters such as weight decay, learning rate, etc. And finally average the predictions of them during inference.
> 
> Hope this can help you.

Hello,Thanks for your open source,I have some question about the pretraned model,did you use these aug method on the newest pretrained model.
